src/Microsoft.Identity.Web/WebApiExtensions/RequiredScopePolicyProvider.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Identity.Web
+{
+    internal class RequiredScopePolicyProvider : IAuthorizationPolicyProvider
+    {
+        public RequiredScopePolicyProvider(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public IConfiguration _configuration;
+        public const string PolicyPrefix = "RequiredScope(";
+
+        public Task<AuthorizationPolicy> GetDefaultPolicyAsync()
+        {
+#pragma warning disable CS8625 // Cannot convert null literal to non-nullable reference type.
+            return Task.FromResult<AuthorizationPolicy>(null);
+#pragma warning restore CS8625 // Cannot convert null literal to non-nullable reference type.
+        }
+
+        public Task<AuthorizationPolicy> GetFallbackPolicyAsync()
+        {
+#pragma warning disable CS8625 // Cannot convert null literal to non-nullable reference type.
```

Comment:
This pragma might need to be conditional. I seems that in .net 5.0 and higher, the return type of the method is `Task<AuthorizationPolicy?>`